lib/containers/utils.pm

```diff
@@ -164,11 +168,16 @@ sub build_img {
     my $runtime = shift;
     die "You must define the runtime!" unless $runtime;
     my $registry = get_var('REGISTRY', 'docker.io');
+
     assert_script_run("cd $dir");
-    record_info('Dockerfile', script_output('cat ./BuildTest/Dockerfile'));
-    assert_script_run("$runtime image pull $registry/library/python:3", timeout => 300);
-    assert_script_run("$runtime tag $registry/library/python:3 python:3");
-    assert_script_run("$runtime build -t myapp BuildTest");
+    if ($runtime =~ /docker|podman/) {
+        assert_script_run("$runtime image pull $registry/library/python:3", timeout => 300);
+        assert_script_run("$runtime tag $registry/library/python:3 python:3");
+        assert_script_run("$runtime build -t myapp BuildTest");
+    }
+    else {
```

Comment:
Since `bud` is specific for `buildah`, why not do a `elsif` here?

    elsif ($runtime =~ /buildah/) {
      ..
    } else {
      die "Unsupported runtime: $runtime");
    }

Reply:
2c15e0b4c

Reply:
Also Yianni can you try to follow the convention and do
```perl
} else {
```
instead of
```perl
}
else {
```
?